src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+import org.w3c.dom.Node;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private float threshold;
```

Comment:
don't need to keep it as a field, calculate it right away when needed

Reply:
not fixed

Reply:
remove this field

Reply:
not fixed